include/nbl/video/utilities/IGPUObjectFromAssetConverter.h

```diff
@@ -120,24 +121,61 @@ class IGPUObjectFromAssetConverter
             // @sadiuk put here more parameters if needed
 
             SPerQueue perQueue[EQU_COUNT];
+            
+            inline void beginCommandBuffer(E_QUEUE_USAGE queueUsage)
+            {
+                uint32_t queue = static_cast<uint32_t>(queueUsage);
+                assert(perQueue[queue].cmdbuf);
```

Comment:
guard with an `if` because its perfectly fine to optionally give cmdbuffers (like I dont want to have to specify all 4 commandbuffers if I only want to make a GPU Pipeline from a CPU pipeline)

Reply:
done